CMakeLists.txt

```diff
@@ -291,7 +291,8 @@ if(OCOS_ENABLE_PYTHON)
   if(NOT "${Python3_FIND_REGISTRY}" STREQUAL "NEVER")
     message(FATAL_ERROR "Python3_FIND_REGISTRY is not NEVER")
   endif()
-  find_package(Python3 COMPONENTS Interpreter Development.Module NumPy)
```

Comment:
check here: https://cmake.org/cmake/help/latest/module/FindPython.html 

it's some standard symbol, I don't know if there is anything wrong with your environment.